config/cloudpaks/cp-shared/operators/templates/0050-sync-cluster-scoped-operators.yaml

```diff
@@ -67,11 +67,22 @@ spec:
                 source: ibm-operator-catalog
                 sourceNamespace: openshift-marketplace
               EOF
-                      sleep 10
-                      oc wait Subscription.operators.coreos.com ibm-cert-manager-operator -n ${IBM_CERT_MANAGER_NAMESPACE:?} --for=CatalogSourcesUnhealthy=False \
-                      && oc wait Subscription.operators.coreos.com ibm-cert-manager-operator -n ${IBM_CERT_MANAGER_NAMESPACE:?} --for=jsonpath='state'="AtLatestKnown" \
-                      && echo "INFO: Successfully installed IBM Cert Manager." \
-                      || result=1
+
+                      while [ $SECONDS -lt 300 ]; do
+                          oc wait Subscription.operators.coreos.com ibm-cert-manager-operator \
+                              -n ${IBM_CERT_MANAGER_NAMESPACE:?} \
+                              --for=condition=CatalogSourcesUnhealthy=False \
+                          && oc wait Subscription.operators.coreos.com ibm-cert-manager-operator \
+                              -n ${IBM_CERT_MANAGER_NAMESPACE:?} \
+                              --for=jsonpath='{.status.state}'="AtLatestKnown" \
+                          && echo "INFO: Successfully installed IBM Cert Manager." \
+                          && result=0 \
+                          && break \
+                          || result=1
+
+                          echo "INFO: Waiting for the subscription to be ready."
+                          sleep 20
+                      done
```

Comment:
Bug fix, found while validating this PR. The condition and JSON paths were wrong. The job would always work on the Job retry because Cert Manager installation eventually worked, but there would always be a failed pod in the cluster post installation.